mobile-app/cypress/integration/functional/wallet/loans/loans.spec.ts

```diff
@@ -585,39 +594,59 @@ context('Wallet - Loans Payback Non-DUSD Loans', () => {
     cy.getByTestID('text_resulting_loan_amount').contains('0.00000000')
     cy.getByTestID('text_resulting_loan_amount_suffix').contains('dTU10')
     cy.getByTestID('text_vault_id').contains(vaultId)
-    // cy.getByTestID('text_current_collateral_ratio').contains('N/A')
+    cy.getByTestID('text_current_collateral_ratio').contains('N/A')
     cy.getByTestID('button_confirm_payback_loan').click().wait(4000)
     cy.getByTestID('txn_authorization_description')
       .contains('Paying 12.00000000 dTU10')
     cy.closeOceanInterface()
     cy.wait(3000)
-    cy.checkVaultTag('READY', VaultStatus.Ready, 'vault_card_1_status', walletTheme.isDark)
+    cy.checkVaultTag('READY', VaultStatus.Ready, 'vault_card_0_status', walletTheme.isDark)
   })
 
   it('should have 0 available dTU10', () => {
     sendTokenToRandomAddress('13', true) // Empty out dTU10 in wallet
     cy.getByTestID('bottom_tab_loans').click()
-    cy.getByTestID('vault_card_1_manage_loans_button').click()
+    cy.getByTestID('vault_card_0_manage_loans_button').click()
     cy.getByTestID('button_browse_loans').should('exist')
   })
 
   it('should borrow dTU10 loan to be paid with DUSD', () => {
     borrowFirstLoan('dTU10', '3')
-    cy.getByTestID('vault_card_1_manage_loans_button').click()
+    cy.getByTestID('vault_card_0_manage_loans_button').click()
     cy.getByTestID('loan_card_dTU10_payback_loan').click()
   })
 
-  it('should display 50% of loan amount if 50% button is pressed with 0 DUSD', () => {
+  /* Paying dTU10 with DUSD balance */
+  it('should display 0 if MAX/50% button is pressed with 0 DUSD', () => {
     cy.getByTestID('payment_token_card_DUSD').click()
-    validate50PercentButton('dTU10', 'DUSD')
+    validateMaxButton('dTU10', 'DUSD', 10, false)
+    validate50PercentButton('dTU10', 'DUSD', 1)
```

Comment:
i noticed some places you passed `hasSufficientBalance = false`, and some you dont, is it intentional? if not, let's make it consistent

Reply:
Yep intention, I added a default . Sure, will update

Reply:
Updated!